davinci/src/main/kotlin/com/pingidentity/davinci/collector/Form.kt

```diff
@@ -39,6 +39,18 @@ internal object Form {
                 collectors.addAll(CollectorFactory.collector(array))
             }
         }
+
+        //Populate values for collectors
+        json["formData"]?.jsonObject?.let { formData ->
+            formData["value"]?.jsonObject?.let { value ->
+                collectors.filterIsInstance<FieldCollector>().forEach { collector ->
+                    value[collector.key]?.let {
+                        collector.init(it)
+                    }
+                }
+            }
+        }
+
```

Comment:
we can make it simple

is this same as above ?

json["formData"]?.jsonObject?.get("value")?.jsonObject?.let { value ->
    collectors.filterIsInstance<FieldCollector>().forEach { collector ->
        value[collector.key]?.let(collector::init)
    }
}

Reply:
Updated.